test/test_psutil.py

```diff
@@ -204,7 +204,7 @@ def sh(cmdline, stdout=subprocess.PIPE, stderr=subprocess.PIPE):
     if stderr:
         warn(stderr)
     if PY3:
-        stdout = str(stdout, sys.stdout.encoding)
+        stdout = str(out, sys.stdout.encoding or sys.getfilesystemencoding() or 'utf-8')
```

Comment:
Sorry, I took a look as I got pinged and was curious about where the problem arose here.

It looks like this change from `stdout` to `out` was unintentional.